packages/esm-service-queues-app/src/patient-queue-metrics/clinic-metrics.component.tsx

```diff
@@ -26,6 +27,7 @@ function ClinicMetrics() {
 
   const currentQueueLocation = useSelectedQueueLocationUuid();
   const { queues } = useQueues(currentQueueLocation);
+  const uniqueQueues = unionBy(queues ?? [], ({ service }) => service?.uuid);
```

Comment:
I'm guessing we need this because there can be multiple queues for the same service?